pkg/fanal/analyzer/analyzer.go

```diff
@@ -146,6 +146,7 @@ type PostAnalysisInput struct {
 type AnalysisOptions struct {
 	Offline      bool
 	FileChecksum bool
+	CollectFiles bool
```

Comment:
We can collect the files by default. I mean we may not need the option. What do you think?

Reply:
:+1: 
I updated the PR without the option